components/pedigree/resources/src/main/resources/pedigree/view/person.js

```diff
@@ -1142,6 +1147,22 @@ define([
             return this.getChildlessStatus();
         },
 
+       /**
+        * Returns cancers as an array of cancer objects.
+        *
+        * @return {Array} of cancer objects
+        */
+        getPhenotipsFormattedCancers: function() {
```

Comment:
This is the format that we want to have in Patient JSON, right?

Reply:
I believe so, yes.